### ISSUE TITLE
excluded_iface options

### ISSUE THREAD

**Issue by julcollas** [intelsdi-x/snap-plugin-collector-interface#33]
Colleting metrics for **lo** interface report some empty tag **hardware_addr**. What do you think about exclude some interfaces from the config ?

**Comment by dancyding** [comment]
Yes it's a good suggestion. Just want to discuss, which config are  more suitable, excluded_iface or included_iface to config? For most cases, we could specify the iface name in metrics name space and except for "*" part, do you think a config for exclue_lo would also be an improvement?

**Comment by julcollas** [comment]
For me, included_iface is already ok with dynamic metric
`/intel/procfs/iface/(eth1;ethx)/*`

**exclude_lo** is too much restrictive i think. 

I also think that a boolean config for **exclude_down** could be nice.
